pkg/minio/minio.go

```diff
@@ -368,64 +367,13 @@ func (m *Minio) waitForMinio(ctx context.Context) error {
 	return nil
 }
 
-func (m *Minio) waitForMinioService(ctx context.Context) error {
-	for {
-		service, err := m.Clientset.CoreV1().Services(m.Namespace).Get(ctx, ServiceName, metav1.GetOptions{})
-		if err != nil {
-			return ErrMinioFailedToGetService.Wrap(err)
-		}
-
-		if service.Spec.Type == v1.ServiceTypeLoadBalancer {
-			if len(service.Status.LoadBalancer.Ingress) == 0 {
-				time.Sleep(waitRetry)
-				continue // Wait until the LoadBalancer IP is available
-			}
-		} else if service.Spec.Type == v1.ServiceTypeNodePort {
-			if service.Spec.Ports[0].NodePort == 0 {
-				return ErrMinioNodePortNotSet
-			}
-		} else if len(service.Spec.ExternalIPs) == 0 {
-			return ErrMinioExternalIPsNotSet
-		}
-
-		// Check if Minio is reachable
-		endpoint, err := m.getEndpoint(ctx)
-		if err != nil {
-			return ErrMinioFailedToGetEndpoint.Wrap(err)
-		}
-
-		if err := checkServiceConnectivity(endpoint); err != nil {
-			time.Sleep(waitRetry) // Retry after some seconds if Minio is not reachable
-			continue
-		}
-
-		break // Minio is reachable, exit the loop
-	}
-
-	select {
-	case <-ctx.Done():
-		return ErrMinioTimeoutWaitingForServiceReady
-	default:
-		return nil
-	}
-}
-
-func checkServiceConnectivity(serviceEndpoint string) error {
-	conn, err := net.DialTimeout("tcp", serviceEndpoint, 2*time.Second)
-	if err != nil {
-		return ErrMinioFailedToConnect.WithParams(serviceEndpoint).Wrap(err)
-	}
-	defer conn.Close()
-	return nil // success
-}
```

Comment:
Why did you remove these?

Reply:
because now we have a func in k8s package that provides `waitForService` and does the same thing.